dev/appveyor-install-dependencies.ps1

```diff
@@ -27,11 +27,13 @@ Function InstallR {
 
   $urlPath = ""
   $latestVer = $(ConvertFrom-JSON $(Invoke-WebRequest https://rversions.r-pkg.org/r-release-win).Content).version
+  $latestVer = "4.3.2"
```

Comment:
This modification is not related to this PR, but is caused by another issue: https://github.com/apache/spark/pull/43631. Only in order to temporarily pass `integration appveyor`, I will remove it after it is completed.

Reply:
We can remove it now?

Reply:
Yeah, this URL `https://rversions.r-pkg.org/r-release-win` has returned the correct value, and I will remove it now.